UP_onboarding_system/point_of_sale/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
```

Comment:
Also please add any more fields that u feel can be added in the below models.

Reply:
Ok sure, I added only those which were mentioned in the onboarding guide. I will add few more .